projects/ngx-text-diff/src/lib/ngx-text-diff.component.ts

```diff
@@ -14,13 +14,14 @@ export class NgxTextDiffComponent implements OnInit {
   @Input() right = '';
   @Input() loading = false;
   @Input() diffContentObservable$: Observable<DiffContent>;
```

Comment:
@ABenassi87 - I am not really sure the convention here, but I have never seen an input name that has a `$`.  While I know it is convention for variable names, it feels a bit strange having this on an `@Input()` for some reason.

Also, I don't think it is convention to have the word `Observable` in the variable name.

Action:
1. I think renaming it to something like `diffContent` would be more straight forward to work with.
2. I don't see where this input is included in the `README.md` as an option. - We should add documentation to all available inputs and explain when to use each one.



_I tried to find some references, but did not find anything too solid_:
https://angular.io/guide/rx-library
https://material.angular.io/components/table/api (one variable input appears to be an observable)
https://material.angular.io/components/chips/api